frontend/packages/citizen-frontend/src/applications/editor/verification/ContactInfoSecondGuardianPreschool.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { useTranslation } from '~localization'
+import { ApplicationGuardianAgreementStatus } from '@evaka/lib-common/src/api-types/application/enums'
+
+type Props = {
+  agreementStatus: ApplicationGuardianAgreementStatus | null
+}
+
+export default React.memo(function ContactInfoSecondGuardianPreschool({
+  agreementStatus
+}: Props) {
+  const t = useTranslation()
+  const tLocal = t.applications.editor.verification.contactInfo
+
+  return (
+    <>
+      {agreementStatus === 'AGREED' && (
+        <span>{tLocal.secondGuardian.agreed}</span>
+      )}
+      {(agreementStatus === 'NOT_AGREED' ||
+        agreementStatus === 'RIGHT_TO_GET_NOTIFIED') && (
+        <span>{tLocal.secondGuardian.notAgreed}</span>
+      )}
+      {!agreementStatus && (
+        <span>{tLocal.secondGuardian.noAgreementStatus}</span>
```

Comment:
now in case where the second guardian lives in same address (or doesn't exist) and the agreement is not asked, the result will be:

**Toisen huoltajan tiedot**
Ei tiedossa

which is a bit confusing / misleading. Since nothing is then asked maybe this whole section should be hidden or we should show same info text as on editor side.